docs/usage/quickstart.rst

```diff
@@ -3,112 +3,154 @@
 ================
-To quickly set up a testing cluster using MetalK8s_, you need 3 machines running
-CentOS_ 7.4 to which you have SSH access (these can be VMs). Each machine
-acting as a Kubernetes_ node (all of them, in this example) also need to have at
-least one disk available to provision storage volumes.
+
+To set up a testing cluster quickly using MetalK8s_ requires three machines
+running CentOS_ 7.4 to which you have SSH access (these can be VMs). Each
+machine acting as a Kubernetes_ node (all of them, in this example) must also
+have at least one disk available to provision storage volumes.
 
 .. todo:: Give some sizing examples
 
 .. _MetalK8s: https://github.com/scality/metal-k8s/
 .. _CentOS: https://www.centos.org
 .. _Kubernetes: https://kubernetes.io
 
-Defining an Inventory
----------------------
-To tell the Ansible_-based deployment system on which machines MetalK8s should
-be installed, a so-called *inventory* needs to be provided. This inventory
-contains a file listing all the hosts comprising the cluster, as well as some
-configuration.
+Define an Inventory
+-------------------
+
+To tell the Ansible_-based deployment system which machines MetalK8s shall be
+installed on, you must provide an *inventory*. This inventory contains a file
+that lists all hosts in the cluster, as well as some configuration.
 
 .. _Ansible: https://www.ansible.com
 
-First, create a directory, e.g. :file:`inventory/quickstart-cluster`, in which
-the inventory will be stored. For our setup, we need to create two files. One
-listing all the hosts, aptly called :file:`hosts`:
+To create an inventory:
+
+  1. Create a directory (for example, :file:`inventory/quickstart-cluster`)
+     in which the inventory will be stored.
+
+     .. code:: shell
+
+       $ mkdir inventory
+       $ mkdir inventory/quickstart-cluster
+       $ cd inventory/quickstart-cluster/
+
+  2. Create the hosts file, which contains a listing of all hosts.
+
+     ::
+
+        node-01 ansible_host=10.0.0.1 ansible_user=centos
+        node-02 ansible_host=10.0.0.2 ansible_user=centos
+        node-03 ansible_host=10.0.0.3 ansible_user=centos
+
+        [kube-master]
+        node-01
+        node-02
+        node-03
+
+        [etcd]
+        node-01
+        node-02
+        node-03
+
+        [kube-node]
+        node-01
+        node-02
+        node-03
 
-.. code-block:: ini
+        [k8s-cluster:children]
+        kube-node
+        kube-master
 
-    node-01 ansible_host=10.0.0.1 ansible_user=centos
-    node-02 ansible_host=10.0.0.2 ansible_user=centos
-    node-03 ansible_host=10.0.0.3 ansible_user=centos
+    Change the IP addresses, domain names, and user names to conform to
+    your infrastructure.
 
-    [kube-master]
-    node-01
-    node-02
-    node-03
+ 3. Create a :file:`group_vars` subdirectory in the inventory directory.
 
-    [etcd]
-    node-01
-    node-02
-    node-03
+  - A second file, called :file:`kube-node.yml`, in a :file:`group_vars`
```

Comment:
Mix of numbered listing and unordered listing is a bit confusing.

Reply:
let's agree to disagree on this one, in order to move this one forward